docs/documentation/population_individual.rst

```diff
@@ -35,6 +35,23 @@ There are two types of individuals:
 These classes are thin wrappers around the `Genome` (see :meth:`cgp.Genome`) and
 the `CartesianGraph` (see :meth:`cgp.CartesianGraph`) classes.
 
+Any custom properties of individuals that are set within the objective function are copied to their offspring.
+This, for example, allows recording of variables used during fitness evaluation:
+
+.. code-block:: python
+
+  def objective(individual):
+      individual.fitness = ...
```

Comment:
I would suggest to add some value here to obtain valid Python code. This will enable us to use `doctest` down the road to automatically test these snippets.

Reply:
ok, i will change it, but i would argue that it's pointless to test this snippet, isn't it?